Car_Collision_Visulaization/src/car_collision.cpp

```diff
@@ -1,5 +1,13 @@
+
 #include "car_collision.h"
 
+    Car::Car()
+    {
+        x_        = 0;
+        y_        = 0;
+        velocity_ = 0;
+        angle_    = 0;  
+    }
 
```

Comment:
We don't need this constructor please delete it

Reply:
I need that default constructor to be used in code,if i don,t write a default constructor then compiler gives error.